raster/rt_pg/rtpg_mapalgebra.c

```diff
@@ -78,10 +78,14 @@ Datum RASTER_mapAlgebra2(PG_FUNCTION_ARGS);
 /* ---------------------------------------------------------------- */
 
 /* Quiet warning */
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunknown-pragmas"
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wgnu-variable-sized-type-not-at-end"
+#if defined(__GNUC__)
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wunknown-pragmas"
+#endif
```

Comment:
@Algunenano The GCC directives were added in the CARTO's fork so I added the `__GNUC__` checking for them.

Reply:
In that case we should leave it as it's defined currently in master:
https://github.com/postgis/postgis/blob/master/raster/rt_pg/rtpg_mapalgebra.c#L80-L103

Reply:
:+1:  done